mint_utils.go

```diff
@@ -108,16 +108,23 @@ func tlsToMintConfig(tlsConf *tls.Config, pers protocol.Perspective) (*mint.Conf
 // unpackInitialOrRetryPacket unpacks packets Initial and Retry packets
 // These packets must contain a STREAM_FRAME for the crypto stream, starting at offset 0.
 func unpackInitialPacket(aead crypto.AEAD, hdr *wire.Header, data []byte, version protocol.VersionNumber) (*wire.StreamFrame, error) {
-	unpacker := &packetUnpacker{aead: &nullAEAD{aead}, version: version}
-	packet, err := unpacker.Unpack(hdr.Raw, hdr, data)
+	buf := *getPacketBuffer()
+	buf = buf[:0]
+	defer putPacketBuffer(&buf)
+
+	decrypted, err := aead.Open(buf, data, hdr.PacketNumber, hdr.Raw)
```

Comment:
Is there a reason why you aren't using the (more efficient) in-place decryption here?

Reply:
I don't think there's a reason, but this is not something that this PR introduced. I opened a separate issue for that: #1277. I'd suggest to do the fix separately.

Reply:
SGTM, thx